site/tutorials/snippets/18/install/download-keptnCLI.md

```diff
@@ -0,0 +1,48 @@
+
+
+## Download Keptn CLI
```

Comment:
This looks like a duplicate of https://keptn.sh/docs/install/cli-install/ .  Do you just want to link to the documentation so you don't have to worry about updating this information.  I am about to add some information about installing the CLI to this section (moving in the material that is currently in https://keptn.sh/docs/0.18.x/reference/cli/ ).  It's a judgement call as to how much you want to replicate material from the docs in a tutorial -- it makes the material more readily available to the person doing the tutorial but you pick up maintenance headaches -- so your choice.

Reply:
Yes, you are right. We can use the documentation references. I have followed the other tutorials convention to provide all steps handy. IMO we can take these changes as part of refactoring efforts(we need to change in all tutorials)